oat-sa/extension-tao-qtiprint
# Fix/BSA-226/Fix cut of right border in booklet

Related to: https://oat-sa.atlassian.net/browse/BSA-226

**Problem:**
The text partly cut in the pdf file
**Solution:**
The section `.include` has `width` more than 100%. Fix width and add margin as for `.item`

How to test:

- create a test with shared stimulus
- generate booklet
- download pdf, check the text is properly rendered in the pdf-file
